src/components/ChatWindow/ChatInput/ChatInput.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import SendMessageIcon from '../../../icons/SendMessageIcon';
+import TextareaAutosize from '@material-ui/core/TextareaAutosize';
+import { makeStyles, Theme, useMediaQuery } from '@material-ui/core';
+import { Conversation } from '@twilio/conversations/lib/conversation';
+
+const useStyles = makeStyles({
+  chatInputContainer: {
+    borderTop: '1px solid #e4e7e9',
+    borderBottom: '1px solid #e4e7e9',
+    padding: '1em 1.2em 1em',
+  },
+  textArea: {
+    paddingTop: '0.2em',
```

Comment:
I used this for my screenshots in slack:

```
padding: '0.75em 1em`,
marginTop: `0.4em`
```

But feel free to play with it. `padding: '0.75em 1em` makes the padding identical to the padding on the text inputs you see before you enter a room.

Reply:
i think this looks good and i like how it stays consistent with the inputs on the "login" page